pkg/webhook/certificatehandler.go

```diff
@@ -0,0 +1,284 @@
+package webhook
```

Comment:
Could you refer the implementation in volcano. https://github.com/volcano-sh/volcano/blob/master/installer/helm/chart/volcano/templates/admission.yaml#L117